### PR TITLE
[Auto tests] Save welder.log file for debugging

### PR DESCRIPTION
**Pull request by rtitle**
In trying to debug [this issue](https://broadworkbench.atlassian.net/browse/IA-1228), it would be helpful to save the welder.log file in the test for debugging.

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by rtitle** [comment]
jenkins retest

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookTestUtils.scala [hunk 1]
```diff
@@ -91,15 +90,11 @@ trait NotebookTestUtils extends LeonardoTestUtils {
...
   }
 
-  implicit val ec: ExecutionContextExecutor = ExecutionContext.global
   def withNewNotebook[T](cluster: Cluster, kernel: NotebookKernel = Python3, timeout: FiniteDuration = 2.minutes)(testCode: NotebookPage => T)(implicit webDriver: WebDriver, token: AuthToken): T = {
     withNotebooksListPage(cluster) { notebooksListPage =>
-      val result: Future[T] = retryUntilSuccessOrTimeout(whenKernelNotReady, failureLogMessage = s"Cannot make new notebook")(30 seconds, 2 minutes) {() =>
-        Future(notebooksListPage.withNewNotebook(kernel, timeout) { notebookPage =>
-          testCode(notebookPage)
-        })
+      notebooksListPage.withNewNotebook(kernel, timeout) { notebookPage =>
```
Comment (rtitle) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookTestUtils.scala:95]: I haven't proved this but I suspect this Future-based retry wasn't playing nice with selenium WebDriver. Tests were failing around this `withNewNotebook` call. Removing the Future & retry to see if the situation improves.

Comment (jdcanas) [reply automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookTestUtils.scala:95]: Were you ever able to reproduce the issues Gary saw?

**Review by jdcanas — APPROVED** [review]

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookTestUtils.scala (hunk 1 above)
Comment (rtitle) [reply automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookTestUtils.scala:95]: I wasn’t - I ran tests many times and they all passed for me. :(

**Pull request merged**